static/admin/config.yml

```diff
@@ -46,7 +46,7 @@ collections:
     folder: "src/markdown-pages/about"
     create: true
     delete: true
-    slug: "{{slug}}" # (    o_o) slug
+    slug: "{{fields.slug}}" # (    o_o) slug
```

Comment:
This appears to only affect pages in the about section - however the problem we're experiencing is with guests. Can we move this to the guests section of the config file?

Reply:
oh yeah i put in the wrong section, will fix